README.md

```diff
@@ -70,6 +70,16 @@ Ways to build `cabal-install` for everyday use
     if you don't have a pre-existing version of `cabal-install`,
     look into the [`bootstrap`](bootstrap) directory.
 
+Learn how to use `cabal` and support
```

Comment:
```suggestion
Learn how to use `cabal` and get support
```
@geekosaur would know better but I think the current phrasing is a bit clumsy in that it reads more like:

```
Learn [the following things]:

1. how to use `cabal`
2. support
```

Reply:
Agreed